src/logic/safe/store/actions/fetchSafe.ts

```diff
@@ -87,9 +87,8 @@ export const buildSafe = async (
     currentVersion: currentVersion ?? '',
     needsUpdate,
     featuresEnabled,
-    balances: localSafe?.balances || Map(),
+    balances: [],
```

Comment:
We are now ignoring localSafe balances here

Reply:
we are not using anymore localStorage, it was used before when mage tokens was a feature.